src/slides/slides.css

```
@@ -555,6 +546,13 @@ body:lang(en) {
   color: unset;
 }
 
+.reveal .slides section section .version {
+  color: var(--grey);
+  right: 50px;
+  font-size: 13px;
```

```suggestion
  right: 50px;
```
Font size and color are already set for all footers on lines 523 and 524.